### ISSUE TITLE
make it possible to create report file names as timestamp

### ISSUE THREAD

**Issue by gjemp** [peerigon/xunit-file#20]
If i do not want to  overwrite the file after every run then at the moment there is no good way to solve the problem. My suggestion is to add new key- value to json that defines if I wanna use timestamp file names or not.

json  should have :

```
{
    "file" : "xunit.xml",
    "timestamp":true,
    "consoleOutput" : {
      "suite" : true,
      "test" : true,
      "fail" : false
    }
}
```

and  code change :

```
/**
 * Module dependencies.
 */
var Date = global.Date
    , setTimeout = global.setTimeout
    , setInterval = global.setInterval
    , clearTimeout = global.clearTimeout
    , clearInterval = global.clearInterval;

var mocha = require("mocha")
  , Base = mocha.reporters.Base
  , utils = mocha.utils
  , escape = utils.escape
  , config = require("../config.json")
  , fs = require("fs")
  , path = require("path")
  , mkdirp = require("mkdirp")
  , filePath = config.timestamp === true ? process.cwd() + "/"+(new Date()).getTime()+".xml" : process.env.XUNIT_FILE || config.file ||   process.cwd() + "/xunit.xml"
  , consoleOutput = config.consoleOutput || {};
```

**Comment by matthaias** [comment]
@gjemp is it solved with https://github.com/peerigon/xunit-file/pull/21? You can now save the file with a timestamp, e.g. `"file": "${cwd}/${timestamp: 'MMDD-hhmm'}/xunit-${pid}.xml"`. For more information about this check the section 'File Path Options' in the README.

**Comment by leomelzer** [comment]
Closing as fixed by #21 

Thanks for the suggestion!
